CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     set(CMAKE_CXX_FLAGS_RELEASE
         "-g -O2"
         CACHE STRING "List of C++ compiler flags for a Release build")
```

Comment:
Why don't you specify c++17 in the release build as well?

Reply:
good point, i will add it